### PR TITLE
Java 11 Experiment

### PR DESCRIPTION
**Pull request by bitwiseman**
* JENKINS issue(s):
    * Issue link or n/a
* Description:
    * ...
* Documentation changes:
    * Link to related jenkins.io PR or explanation for why doc change not needed
* Users/aliases to notify:
    * ...

### CODE REVIEW

**Comment by bitwiseman** [comment]
Build succeeded, only a few tests failed.

File: Jenkinsfile [hunk 1]
```diff
@@ -65,9 +65,9 @@ pipeline {
                         }
                     }
                 }
-                stage("linux-newer-core") {
+                stage("linux-newer-jenkins-core") {
                     agent {
-                        label "highmem"
+                        label "linux"
```
Comment (abayer) [inline Jenkinsfile]: Why change off `highmem`? The `linux` agents are smaller, and Declarative's tests can end up timing out on them.

Comment (bitwiseman) [reply Jenkinsfile]: Trying to use the standard machine config if possible.  I'll revert.

Comment (abayer) [reply Jenkinsfile]: Thanks!

**Pull request merged**